src/main/resources/project/ui_forms/deploy_DE.xml

```diff
@@ -97,6 +97,17 @@
         <dependsOn>connection_config</dependsOn>
     </formElement>
     <formElement>
+        <label>Associate IP?:</label>
+        <property>associate_ip</property>
+        <value/>
+        <documentation>Associate IP to deployed instance?:</documentation>
```

Comment:
Lets add to the tooltip that this may be unchecked for Rackspace, and Rackspace will automatically assign a public IP.